presto-geospatial-toolkit/src/main/java/com/facebook/presto/geospatial/JtsGeometryUtils.java

```diff
@@ -128,12 +190,28 @@ private static Point readPoint(SliceInput input)
         return GEOMETRY_FACTORY.createPoint(coordinates);
     }
 
+    private static void writePoint(Point point, DynamicSliceOutput sliceOutput) {
+        requireNonNull(sliceOutput, "output is null");
+        if (!point.isEmpty()) {
+            sliceOutput.writeInt(EsriShapeType.POINT.code);
+            writeCoordinate(point.getCoordinate(), sliceOutput);
+        } else {
+            throw new IllegalArgumentException("Trying to write an empty point.");
+        }
+    }
+
     private static Coordinate readCoordinate(SliceInput input)
     {
         requireNonNull(input, "input is null");
         return new Coordinate(input.readDouble(), input.readDouble());
     }
 
+    private static void writeCoordinate(Coordinate point, DynamicSliceOutput sliceOutput) {
```

Comment:
s/DynamicSliceOutput/SliceOutput

s/sliceOutput/output for brevity

s/point/coordinate